### ISSUE TITLE
False positive on sendTo

### ISSUE THREAD

**Issue by rlaffers** [rlaffers/eslint-plugin-xstate#22]
**Describe the bug**
The 2.2.0 version of the plugin reports an error xstate/event-names for a correct usage of the `sendTo` action creator:

```javascript
sendTo('anotherActor', { type: 'MY_EVENT' })
```

**Expected behavior**
No error is reported.

**Actual behavior**
Error is reported:
Prefer "ANOTHER_ACTOR" over "anotherActor" event name                                       xstate/event-names

**Versions (please complete the following information):**
 - Node version: 18
 - ESLint version: 8.46
 - eslint-plugin-xstate version: 2.2.0
